playfab-docs/features/Insights/connectivity/creating-AAD-app-for-insights.md

```diff
@@ -64,6 +64,9 @@ Now we will connect the Azure app to your title database.
    * **Analytics data read access**, to run queries and simple management commands.
    * **Analytics data write access**, to create/drop tables, alter retention policy, ingest data, purge.
 
+> [!NOTE] 
+> Changing the roles for the application is only possible through Roles -> Edit Role Members Menu if you don't want to assign an Email Address to your application 
```

Comment:
"email address" should be lowercase.

bold the menu UI text.  **Roles**>**Edit Role Members** if you don't want to assign.....

Reply:
Done!